src/components/common/Questionnaire/ExpandedModal/ScoreRange/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import styles from './ScoreRange.module.css';
+
+const scoreRange = {
+  min: 0,
+  max: 10,
+};
+
+const ScoreRange = ({ handleUserFeedback }) => {
+  const [score, setScore] = useState(null);
+  const handleScoreClick = e => {
+    const newScore = Number(e.target.textContent);
+    setScore(newScore);
+    handleUserFeedback(newScore);
+  };
+  const scoreButtons = Array.from({ length: scoreRange.max + 1 }, (_, i) => (
+    <button
+      className={`${styles.button} ${score === i ? styles.activeButton : ''}`}
```

Comment:
Suggestion: 可以使用 `classnames` 這個套件

```js
import cn from 'classnames'
```
```suggestion
      className={cn(styles.button, { [styles.activeButton]: score === i })}
```

Reply:
已修改～